pkgs/by-name/wa/watchyourlan/package.nix

```
@@ -0,0 +1,40 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  makeBinaryWrapper,
+  lib,
+  arp-scan,
+}:
+buildGoModule rec {
+  pname = "watchyourlan";
+  version = "2.0.4";
+
```

```suggestion

  src = fetchFromGitHub {
    owner = "aceberg";
    repo = "WatchYourLAN";
    rev = version;
    hash = "sha256-nJYDGCkT8vCkxySLONG3OkWVkaBqXqSFgd7N1TTMAf4=";
  };

```
we usually order things like this

---

Done.